Backend/sub_crates/validator/proptest-regressions/tests/password.txt

```diff
@@ -5,3 +5,4 @@
 # It is recommended to check this file in to source control so that
 # everyone who runs the test benefits from these saved cases.
 cc c3b98dff6674ece1c38d4b2ce45bf131673a95f1381b4ccc4895b34ceebe803c # shrinks to pass = "০𐝀 🞀"
+cc 610bfdc5cd06276d4dedcac3ea4a12792efa50b2f59e74d0426dcd1c47069451 # shrinks to pass = "𞠀ઽ𑈀a"
```

Comment:
So apparently this input still counts as alphanumeric 😱 Or I messed something up

Reply:
Yep. isses auch